bpf/bpf_lb.c

```diff
@@ -75,7 +75,11 @@ static inline int handle_ipv6(struct __sk_buff *skb)
 	nexthdr = ip6->nexthdr;
 	ipv6_addr_copy(&key.address, (union v6addr *) &ip6->daddr);
 	l3_off = ETH_HLEN;
-	l4_off = ETH_HLEN + ipv6_hdrlen(skb, ETH_HLEN, &nexthdr);
+	hdrlen = ipv6_hdrlen(skb, ETH_HLEN, &nexthdr);
+	if (IS_ERR(hdrlen))
```

Comment:
Patch generally looks fine. IS_ERR() would not be required in this context though, only testing for '< 0' should be fine for catching the DROP_* error codes. Could you still fix this up? Otherwise looks good.

Reply:
Thanks for reviewing, updated.